src/chrome_bindings/server.js

```diff
@@ -28,6 +28,7 @@ const server = http.createServer(async (req, res) => {
   const contentType = {
     html: 'text/html',
     js: 'application/javascript',
+    map: 'application/json',
```

Comment:
Because otherwise it failed with exception once trying to load `.js.map` files:

```
> node dist/web/server.js

Server is running on: http://localhost:8000
./on-chain-state-mgmt-zkapp-ui.html
./on-chain-state-mgmt-zkapp-ui.js
./index.js
./examples/zkapps/hello_world/hello_world.js
./e2eTestsHelpers.js
./favicon.ico
./favicon.ico
./examples/zkapps/hello_world/hello_world.js.map
node:internal/errors:465
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_HTTP_INVALID_HEADER_VALUE]: Invalid value "undefined" for header "content-type"
    at storeHeader (node:_http_outgoing:519:5)
    at processHeader (node:_http_outgoing:514:3)
    at ServerResponse._storeHeader (node:_http_outgoing:411:11)
    at ServerResponse.writeHead (node:_http_server:348:8)
    at Server.<anonymous> (file:///./snarkyjs/dist/web/server.js:34:7) {
  code: 'ERR_HTTP_INVALID_HEADER_VALUE'
```